protected/controllers/PatientController.php

```diff
@@ -1668,6 +1711,7 @@ public function actionUpdate($id)
         $practice = isset($patient->practice) ? $patient->practice : new Practice();
         $practicecontact = isset($patient->practice) ? $patient-> practice->contact : new Contact();
         $practiceaddress = isset($practicecontact) && isset($practicecontact->address) ? $practicecontact->address : new Address();
+        $patientuserreferral = isset($patient->patientuserreferral[0]) ? $patient->patientuserreferral[0] : new PatientUserReferral();
```

Comment:
@marshl Before you comment, I had @prathyushaSama add an order condition to the relationship so element 0 will always be the most recently added researcher.